packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2602,8 +2622,8 @@ export default mixins(
 			}
 			this.historyStore.reset();
 			this.uiStore.nodeViewInitialized = true;
-			document.addEventListener('keydown', this.keyDown);
-			document.addEventListener('keyup', this.keyUp);
+			document.addEventListener('keydown', this.keyDown, { capture: true, once: false });
+			document.addEventListener('keyup', this.keyUp, { capture: true, once: false });
```

Comment:
Setting `capture: true` here breaks keyboard shortcuts in NodeCreator panel, for example CMD-A, CMD-C, CMD-V